### PR TITLE
fix(commits): Restore commit links

### PR DESCRIPTION
**Pull request by MeredithAnya**
checks for the new repo provider id

### CODE REVIEW

File: src/sentry/static/sentry/app/components/commitLink.jsx [hunk 1]
```diff
@@ -13,10 +13,16 @@ class CommitLink extends React.Component {
 
   getCommitUrl = () => {
     // TODO(jess): move this to plugins
-    if (this.props.repository.provider.id === 'github') {
+    if (
+      ['github', 'integrations:github'].indexOf(this.props.repository.provider.id) != -1
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/commitLink.jsx]: You can use [`includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)

File: src/sentry/static/sentry/app/components/commitLink.jsx [hunk 2]
```diff
@@ -38,16 +44,22 @@ class CommitLink extends React.Component {
...
-        )}
-        {repository.provider.id == 'bitbucket' && (
-          <InlineSvg
-            src="icon-bitbucket"
-            style={{verticalAlign: 'text-top'}}
-            size="14px"
-          />
-        )}
+        {repository.provider.id == 'github' ||
+          (repository.provider.id == 'integrations:github' && (
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/commitLink.jsx]: Why not do the same thing here? `['github', 'integrations:github'].includes(repository.provider.id) && (`

**Review by evanpurkhiser — APPROVED** [review]
Left a couple minor style comments. Looks good!

File: src/sentry/static/sentry/app/components/commitLink.jsx [hunk 3]
```diff
@@ -13,10 +13,16 @@ class CommitLink extends React.Component {
...
     // TODO(jess): move this to plugins
-    if (this.props.repository.provider.id === 'github') {
+    if (
+      ['github', 'integrations:github'].indexOf(this.props.repository.provider.id) != -1
+    ) {
       return this.props.repository.url + '/commit/' + this.props.commitId;
     }
-    if (this.props.repository.provider.id === 'bitbucket') {
+    if (
+      ['bitbucket', 'integrations:bitbucket'].indexOf(
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/commitLink.jsx]: I think you can instead use `Array.prototype.includes` ([MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)) to do the same thing without the `-1` check.

File: src/sentry/static/sentry/app/components/commitLink.jsx [hunk 4]
```diff
@@ -38,16 +44,22 @@ class CommitLink extends React.Component {
...
-          <InlineSvg src="icon-github" style={{verticalAlign: 'text-top'}} size="14px" />
-        )}
-        {repository.provider.id == 'bitbucket' && (
-          <InlineSvg
-            src="icon-bitbucket"
-            style={{verticalAlign: 'text-top'}}
-            size="14px"
-          />
-        )}
+        {repository.provider.id == 'github' ||
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/components/commitLink.jsx]: If we're repeating this check all over might be worth having a `isBitbucket` helper method.

```javascript
const isBitbucket = (providerId) => ['bitbucket', 'bitbucket:integrations'].includes(providerId);
```

**Pull request merged**